Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -620,11 +627,25 @@ - (BranchLinkProperties *)getFirstReferringBranchLinkProperties {
 }
 
 - (NSDictionary *)getFirstReferringParams {
-    return [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
+    NSDictionary *origInstallParams = [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
```

Comment:
oh, that's odd. I guess I'm okay with that if @aaustin is okay with it.
I'll test it manually now

On Mon, Jan 4, 2016 at 1:21 PM, Dan Walkowski <notifications@github.com>
wrote:

> In Branch-SDK/Branch-SDK/Branch.m
> <https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/263#discussion_r48768876>
> :
>
> > @@ -620,11 +627,25 @@ - (BranchLinkProperties *)getFirstReferringBranchLinkProperties {
> >  }
> >
> >  - (NSDictionary *)getFirstReferringParams {
> > -    return [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
> > +    NSDictionary *origInstallParams = [BNCEncodingUtils decodeJsonStringToDictionary:self.preferenceHelper.installParams];
>
> That's not correct. We always need to return the decoded original params.
> The if block in the middle just adds more debug params if there are any.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BranchMetrics/iOS-Deferred-Deep-Linking-SDK/pull/263/files#r48768876>
> .
>



-- 
[image: Branch] <https://branch.io/?bmp=xink-sig>
Derrick Staten
Engineer
What is deep linking? <https://t.xink.io/Tracking/Index/Kg4AAGZBAABoTAYA0>
Check out our webinars <https://t.xink.io/Tracking/Index/Kw4AAGZBAABoTAYA0>